web/src/main/java/org/springframework/security/web/authentication/preauth/AbstractPreAuthenticatedProcessingFilter.java

```diff
@@ -348,6 +326,14 @@ public void setAuthenticationFailureHandler(AuthenticationFailureHandler authent
 		this.authenticationFailureHandler = authenticationFailureHandler;
 	}
 
+	/**
+	 * Sets the request matcher to check whether to proceed the request further.
+	 */
+	public void setAuthenticationRequestMatcher(RequestMatcher authenticationRequestMatcher) {
```

Comment:
rename to setRequiresAuthenticationRequestMatcher to align with AbstractAuthenticationProcessingFilter